### PR TITLE
Add superdataset replicas number config

### PR DESCRIPTION
**Pull request by EvanLjp**
…set index.

Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Improve performance

- Related issues

___
### New feature or improvement
- Describe the details and related test reports.
mertics or common record index  would improve the stability of index and improve search performance when the replicas number is greater than 0.
But superdataset index would be cost so many resources when the replicas number is same as the number of above indices.
So distinct the config may be a good choice.

### CODE REVIEW

File: oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchConfig.java [hunk 1]
```diff
@@ -38,6 +39,12 @@
     private int superDatasetIndexShardsFactor = 5;
     @Setter
     private int indexReplicasNumber = 0;
+    /**
+     * @since 8.2.0, the replicas number is for super size dataset record replicas number
+     * @see SuperDataset
+     */
+    @Setter
+    private int superDatasetIndexReplicasNumber = 0;
```
Comment (wu-sheng) [inline oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchConfig.java:47]: As you moved all superset config in one place in the doc and config file, please do the same thing for the config class

Comment (EvanLjp) [reply oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchConfig.java:51]: done

File: docs/en/setup/backend/configuration-vocabulary.md [hunk 2]
```diff
@@ -74,12 +74,13 @@ core|default|role|Option values, `Mixed/Receiver/Aggregator`. **Receiver** mode
...
-| - | - | superDatasetIndexShardsFactor | Super data set has been defined in the codes, such as trace segments. This factor provides more shards for the super data set, shards number = indexShardsNumber * superDatasetIndexShardsFactor. Also, this factor effects Zipkin and Jaeger traces.|SW_STORAGE_ES_SUPER_DATASET_INDEX_SHARDS_FACTOR|5 |
 | - | - | indexReplicasNumber | Replicas number of new indexes | SW_STORAGE_ES_INDEX_REPLICAS_NUMBER | 0 |
+| - | - | superDatasetDayStep | Represent the number of days in the super size dataset record index, the default value is the same as dayStep when the value is less than 0.|SW_SUPERDATASET_STORAGE_DAY_STEP|-1 |
+| - | - | superDatasetIndexShardsFactor | Super data set has been defined in the codes, such as trace segments. This factor provides more shards for the super data set, shards number = indexShardsNumber * superDatasetIndexShardsFactor. Also, this factor effects Zipkin and Jaeger traces.|SW_STORAGE_ES_SUPER_DATASET_INDEX_SHARDS_FACTOR|5 |
+| - | - | superDatasetIndexReplicasNumber | Represent the replicas number in the super size dataset record index.|SW_STORAGE_ES_SUPER_DATASET_INDEX_REPLICAS_NUMBER|0 |
```
Comment (wu-sheng) [inline docs/en/setup/backend/configuration-vocabulary.md:83]: As usual, don't change the default value, it used to be 1 replicate. Many new users even don't expect this.

Comment (EvanLjp) [reply docs/en/setup/backend/configuration-vocabulary.md:83]: image
the config is change to 0 at 2020-03-15

Comment (EvanLjp) [reply docs/en/setup/backend/configuration-vocabulary.md:83]: i just sync  real config to doc

Comment (wu-sheng) [reply docs/en/setup/backend/configuration-vocabulary.md:83]: Got it.

**Review by wu-sheng — APPROVED** [review]
LGTM.

**Pull request merged**